app/FAIRshakeWeb/view.py

```diff
@@ -120,59 +121,65 @@ def new_evaluation(repository: RepositoryAPI, rubric: RubricAPI, assessment: Ass
     rubric_ids=json.loads(request.form.get('rubric_ids'))
     rubrics=[rubric.get(id=rubric_id)[0] for rubric_id in rubric_ids]
     for rubric_index, rubric in enumerate(rubrics):
-      criteria = [] # evaluations
-      for criterion_index, criterion in enumerate(rubric.criteria): # questions
-        criteria.append(
-          {
-            'name': criterion.name,
-            'value': request.form.get('q-{:d}.{:d}'.format(rubric_index,criterion_index))
-          }
+      criteria = [
+        {
+          'id': criterion.id,
+          'value': request.form.get('q-{:d}.{:d}'.format(rubric_index, criterion_index))
+        }
+        for criterion_index, criterion in enumerate(rubric.criteria)
+      ]
+      assessment.post(
+        AssessmentModel(
+        object=resource_id,
+        user=current_user['id'],
+        rubric=rubric.id,
+        criteria=criteria
         )
-      assessmentModel = {
-        'object': resource_id,
-        'user': current_user['id'],
-        'rubric': rubric.id,
-        'criteria': criteria
-      }
-      assessment.post(json.dumps(assessmentModel)) # post consumes json, assessmentmodel representation
-    project=repository.get(resource_id).tag
+      )
+    project=repository.get(resource_id).tags[0]
```

Comment:
The first tag has no special meaning--we should instead search for the specific tag associated with the project--it can be delineated by some encoding e.g. `project:5`.

Reply:
I will address this